ecc/bls12-381/g1_test.go

```diff
@@ -872,6 +872,10 @@ func fuzzCofactorOfG1Jac(f fp.Element) G1Jac {
 	return res
 }
 
+func GeneratePointNotInG1(f fp.Element) G1Jac {
```

Comment:
These must not be declared in the `_test.go` files. Those are only compiled when testing the package they are part of, and not part of the externally accesible set

Reply:
true! my bad